apps/studio/pages/api/platform/projects/[ref]/analytics/endpoints/[name].ts

```diff
@@ -23,15 +24,26 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       }
 
     default:
-      res.setHeader('Allow', ['GET'])
+      res.setHeader('Allow', ['GET', 'POST'])
       res.status(405).json({ data: null, error: { message: `Method ${method} Not Allowed` } })
   }
 }
 
 const proxyRequest = async (req: NextApiRequest) => {
   const { name, ...toForward } = req.query
-  const payload = { ...toForward, project_tier: 'ENTERPRISE' }
-  const search = '?' + new URLSearchParams(payload as any).toString()
+  const project_tier = 'ENTERPRISE'
```

Comment:
need const for plan name

Reply:
this is just a hard code for local/self-host studio so not super important tbh